esmvalcore/preprocessor/_io.py

```diff
@@ -142,43 +142,73 @@ def _fix_cube_attributes(cubes):
         cube.attributes = attributes
 
 
-def concatenate(cubes):
-    """Concatenate all cubes after fixing metadata."""
-    _fix_cube_attributes(cubes)
-
-    concatenated = iris.cube.CubeList(cubes).concatenate()
-    if len(concatenated) == 2:
-        try:
-            concatenated[0].coord('time')
-            concatenated[1].coord('time')
-        except iris.exceptions.CoordinateNotFoundError:
-            pass
+def _by_two_concatenation(concatenated):
```

Comment:
```suggestion
def _by_two_concatenation(cubes):
```
would be a nicer name than concatenated for the cubelist coming in, because it is not yet concatenated